arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2667,6 +2667,10 @@
           "type": "boolean",
           "description": "Gets or sets whether the extension handler should be automatically upgraded across minor versions."
         },
+        "forceUpdateTag": {
+          "type": "string",
+          "description": "Gets or sets whether the extension handler should be forced to re-run even if the extension configuration has not changed."
```

Comment:
One minor change "Gets or sets whether the extension handler should be forced to re-run even the extension if the extension configuration has not changed."

Why Get? The statement doesn't make sense in terms of a Get